### PR TITLE
Add information to run besu

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

File: docs/dev/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+## Mining locally
+
+If you would like to mine on the network, you will need to install a miner to connect to Besu (see below).
+
+You will need to enable Besu's stratum mining as well:
+
+1. `$ docker run -p 8545:8545 tmio/besu-keccak --rpc-http-enabled --rpc-http-api=admin,eth,debug,miner,net,txpool,priv,trace,web3 --rpc-http-cors-origins="all" --network=ecip1049_dev --miner-enabled --miner-coinbase=fe3b557e8fb62b89f4916b721be55ceb828dbd73
+  --miner-stratum-enabled --miner-stratum-host=0.0.0.0`
+
+Currently, we have only created a CPU miner in Python, however we will be working on an open source GPU version as well. Please consult [our resources](/mine/) if you are interested in mining.
```
Comment (antsankov) [inline docs/dev/index.md:20]: I tried to get this to work, but whenever I ran the miner I would always get the error: 

web 3 'code': -32000, 'message': 'No mining work available yet'

I would mention that the auxilariy Python CPU miner is useful for learning purposes but is not working right now with the testnet (I'd be curious if you got it to work yourself)

File: docs/dev/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+* `8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63`
+* `c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3`
+* `ae6ae8e5ccbfb04590405997ee2d52d2b330726137b875053c36d94e974d162f`
+
+You can import them into Metamask.
+
+## More difficulty
+
+The dev network uses a fixed, small difficulty so it uses CPU mining by default.
+You can provide a different genesis file with the following approach:
```
Comment (antsankov) [inline docs/dev/index.md:40]: I would include a note, that the test difficulty of `0x10000`  leads to about 2 blocks for second on a basic CPU using the built-in Besu miner.

File: docs/dev/index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+1. `$ docker run -p 8545:8545 tmio/besu-keccak --rpc-http-enabled --rpc-http-api=admin,eth,debug,miner,net,txpool,priv,trace,web3 --rpc-http-cors-origins="all" --network=ecip1049_dev --miner-enabled --miner-coinbase=fe3b557e8fb62b89f4916b721be55ceb828dbd73`
+
+## Mining locally
+
+If you would like to mine on the network, you will need to install a miner to connect to Besu (see below).
+
+You will need to enable Besu's stratum mining as well:
+
+1. `$ docker run -p 8545:8545 tmio/besu-keccak --rpc-http-enabled --rpc-http-api=admin,eth,debug,miner,net,txpool,priv,trace,web3 --rpc-http-cors-origins="all" --network=ecip1049_dev --miner-enabled --miner-coinbase=fe3b557e8fb62b89f4916b721be55ceb828dbd73
+  --miner-stratum-enabled --miner-stratum-host=0.0.0.0`
```
Comment (antsankov) [inline docs/dev/index.md:18]: Can you include a link in the comment above the best place where users can learn more about mining with Besu (stratum, miner-enabled) d

File: docs/dev/index.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+We are making available a simple Docker-based set up with Hyperledger Besu so you can try out locally Keccak mining.
+The local network has a very small fixed difficulty as well so you can easily produce blocks on your machine with CPU mining.
+
+1. `$ docker run -p 8545:8545 tmio/besu-keccak --rpc-http-enabled --rpc-http-api=admin,eth,debug,miner,net,txpool,priv,trace,web3 --rpc-http-cors-origins="all" --network=ecip1049_dev --miner-enabled --miner-coinbase=fe3b557e8fb62b89f4916b721be55ceb828dbd73`
+
+## Mining locally
+
+If you would like to mine on the network, you will need to install a miner to connect to Besu (see below).
+
+You will need to enable Besu's stratum mining as well:
```
Comment (antsankov) [inline docs/dev/index.md:15]: Can you include a link in the comment above the best place where users can learn more about mining with Besu (stratum, miner-enabled)?

File: docs/dev/index.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
+---
+title: Node
+lang: en-US
+---
+## Running locally
+We are making available a simple Docker-based set up with Hyperledger Besu so you can try out locally Keccak mining.
+The local network has a very small fixed difficulty as well so you can easily produce blocks on your machine with CPU mining.
+
+1. `$ docker run -p 8545:8545 tmio/besu-keccak --rpc-http-enabled --rpc-http-api=admin,eth,debug,miner,net,txpool,priv,trace,web3 --rpc-http-cors-origins="all" --network=ecip1049_dev --miner-enabled --miner-coinbase=fe3b557e8fb62b89f4916b721be55ceb828dbd73`
```
Comment (antsankov) [inline docs/dev/index.md:9]: This command works great, and was able to start it up a few times, get running for a few thousand blocks, and then I killed the server with ctrl-c, however everytime I used it after on the machine the command hangs on, for 5+ minutes, and then does nothing. This was the debug logs: 
```
root@ubuntu-s-1vcpu-2gb-nyc3-01:~# docker run -p 8545:8545 tmio/besu-keccak --rpc-http-enabled --rpc-http-api=admin,eth,debug,miner,net,txpool,priv,trace,web3 --rpc-http-cors-origins="all"  --miner-enabled --miner-coinbase=fe3b557e8fb62b89f4916b721be55ceb828dbd73 --genesis-file=genesis.json --logging=DEBUG
Setting logging level to DEBUG
2021-02-18 05:49:08.277+00:00 | main | DEBUG | Native | Looking in classpath from jdk.internal.loader.ClassLoaders$AppClassLoader@627551fb for /com/sun/jna/linux-x86-64/libjnidispatch.so
2021-02-18 05:49:08.285+00:00 | main | DEBUG | Native | Found library resource at jar:file:/opt/besu/lib/jna-5.6.0.jar!/com/sun/jna/linux-x86-64/libjnidispatch.so
2021-02-18 05:49:08.296+00:00 | main | DEBUG | Native | Extracting library to /opt/besu/.cache/JNA/temp/jna9925455032403739316.tmp
2021-02-18 05:49:08.300+00:00 | main | DEBUG | Native | Trying /opt/besu/.cache/JNA/temp/jna9925455032403739316.tmp
2021-02-18 05:49:08.303+00:00 | main | DEBUG | Native | Found jnidispatch at /opt/besu/.cache/JNA/temp/jna9925455032403739316.tmp
2021-02-18 05:49:08.351+00:00 | main | DEBUG | NativeLibrary | Looking for library 'eth_pairings'
2021-02-18 05:49:08.352+00:00 | main | DEBUG | NativeLibrary | Adding paths from jna.library.path: null
2021-02-18 05:49:08.357+00:00 | main | DEBUG | NativeLibrary | Trying libeth_pairings.so
2021-02-18 05:49:08.361+00:00 | main | DEBUG | NativeLibrary | Loading failed with message: libeth_pairings.so: cannot open shared object file: No such file or directory
2021-02-18 05:49:08.362+00:00 | main | DEBUG | NativeLibrary | Adding system paths: [/usr/lib/x86_64-linux-gnu, /lib/x86_64-linux-gnu, /lib64, /usr/lib, /lib, /usr/local/openjdk-11/lib, /usr/local/openjdk-11/lib/server, /usr/local/openjdk-11/lib/jli]
2021-02-18 05:49:08.363+00:00 | main | DEBUG | NativeLibrary | Trying libeth_pairings.so
2021-02-18 05:49:08.369+00:00 | main | DEBUG | NativeLibrary | Loading failed with message: libeth_pairings.so: cannot open shared object file: No such file or directory
2021-02-18 05:49:08.370+00:00 | main | DEBUG | NativeLibrary | Looking for version variants
2021-02-18 05:49:08.372+00:00 | main | DEBUG | Native | Looking in classpath from jdk.internal.loader.ClassLoaders$AppClassLoader@627551fb for eth_pairings
2021-02-18 05:49:08.377+00:00 | main | DEBUG | Native | Found library resource at jar:file:/opt/besu/lib/bls12-381-0.3.0.jar!/linux-x86-64/libeth_pairings.so
2021-02-18 05:49:08.381+00:00 | main | DEBUG | Native | Extracting library to /opt/besu/.cache/JNA/temp/jna17331024606749903939.tmp
2021-02-18 05:49:08.466+00:00 | main | DEBUG | NativeLibrary | Found library 'eth_pairings' at /opt/besu/.cache/JNA/temp/jna17331024606749903939.tmp
2021-02-18 05:49:08.521+00:00 | main | INFO  | AbstractAltBnPrecompiledContract | Using LibEthPairings native alt bn128
2021-02-18 05:49:10.716+00:00 | main | DEBUG | NativeLibrary | Looking for library 'secp256k1'
2021-02-18 05:49:10.718+00:00 | main | DEBUG | NativeLibrary | Adding paths from jna.library.path: null
2021-02-18 05:49:10.719+00:00 | main | DEBUG | NativeLibrary | Trying libsecp256k1.so
2021-02-18 05:49:10.721+00:00 | main | DEBUG | NativeLibrary | Loading failed with message: libsecp256k1.so: cannot open shared object file: No such file or directory
2021-02-18 05:49:10.724+00:00 | main | DEBUG | NativeLibrary | Adding system paths: [/usr/lib/x86_64-linux-gnu, /lib/x86_64-linux-gnu, /lib64, /usr/lib, /lib, /usr/local/openjdk-11/lib, /usr/local/openjdk-11/lib/server, /usr/local/openjdk-11/lib/jli]
2021-02-18 05:49:10.725+00:00 | main | DEBUG | NativeLibrary | Trying libsecp256k1.so
2021-02-18 05:49:10.727+00:00 | main | DEBUG | NativeLibrary | Loading failed with message: libsecp256k1.so: cannot open shared object file: No such file or directory
2021-02-18 05:49:10.727+00:00 | main | DEBUG | NativeLibrary | Looking for version variants
2021-02-18 05:49:10.730+00:00 | main | DEBUG | Native | Looking in classpath from jdk.internal.loader.ClassLoaders$AppClassLoader@627551fb for secp256k1
2021-02-18 05:49:10.733+00:00 | main | DEBUG | Native | Found library resource at jar:file:/opt/besu/lib/secp256k1-0.3.0.jar!/linux-x86-64/libsecp256k1.so
2021-02-18 05:49:10.735+00:00 | main | DEBUG | Native | Extracting library to /opt/besu/.cache/JNA/temp/jna9881794633645501678.tmp
2021-02-18 05:49:10.746+00:00 | main | DEBUG | NativeLibrary | Found library 'secp256k1' at /opt/besu/.cache/JNA/temp/jna9881794633645501678.tmp
```

Looks like it's fialing to load some of those .so files, is this related to the PR or just an idiosynchrocy on my end?

Comment (atoulme) [reply docs/dev/index.md:9]: If you want to kill Besu with Ctrl+C, make sure to run Docker with the `-it` flag, so the signal is passed to the Besu process. The logs you point show that the native libs are eventually found and made available, so it's all good there.
For example:
```
2021-02-18 05:49:10.733+00:00 | main | DEBUG | Native | Found library resource at jar:file:/opt/besu/lib/secp256k1-0.3.0.jar!/linux-x86-64/libsecp256k1.so
```

Comment (antsankov) [reply docs/dev/index.md:9]: Fantastic, will do.

**Pull request merged**